pycytominer/tests/test_cyto_utils/test_output.py

```diff
@@ -39,64 +39,88 @@
 ).reset_index(drop=True)
 
 # Set default compression options
-compression_options = {"method": "gzip"}
+TEST_COMPRESSION_OPTIONS = {"method": "gzip"}
 
 
-def test_compress():
+def test_output_default():
 
-    output_filename = os.path.join(tmpdir, "test_compress.csv.gz")
+    output_filename = os.path.join(TMPDIR, "test_compress.csv.gz")
 
-    output(
-        df=data_df,
+    output_result = output(
+        df=DATA_DF,
         output_filename=output_filename,
-        compression_options=compression_options,
+        compression_options=TEST_COMPRESSION_OPTIONS,
         float_format=None,
     )
-    result = pd.read_csv(output_filename)
+    result = pd.read_csv(output_result)
 
     pd.testing.assert_frame_equal(
-        result, data_df, check_names=False, check_exact=False, atol=1e-3
+        result, DATA_DF, check_names=False, check_exact=False, atol=1e-3
     )
 
 
-def test_compress_tsv():
+def test_output_tsv():
     # Test input filename of writing a tab separated file
-    output_filename = os.path.join(tmpdir, "test_compress.tsv.gz")
-    output(
-        df=data_df,
+    output_filename = os.path.join(TMPDIR, "test_compress.tsv.gz")
+    output_result = output(
+        df=DATA_DF,
         sep="\t",
         output_filename=output_filename,
-        compression_options=compression_options,
+        compression_options=TEST_COMPRESSION_OPTIONS,
         float_format=None,
     )
 
-    result = pd.read_csv(output_filename, sep="\t")
+    result = pd.read_csv(output_result, sep="\t")
+    pd.testing.assert_frame_equal(
+        result, DATA_DF, check_names=False, check_exact=False, atol=1e-3
+    )
+
+
+def test_output_parquet():
+    """
+    Tests using output function with parquet type
+    """
+
+    output_filename = os.path.join(TMPDIR, "test_output.parquet")
+
+    # test with base output arguments and
+    # kwargs output arguments for pd.DataFrame.to_parquet
+    output_result = output(
+        df=DATA_DF,
+        output_filename=output_filename,
+        output_type="parquet",
+        compression_options="snappy",
+        engine="auto",
```

Comment:
ah, is this a `kwarg` example? Wait no, what is `engine`? Sorry, I missed this in my review of these updates.

I think we can be opinionated about parquet outputs (am I not thinking about downsides of this?)

Reply:
`engine` came from or was for a `pd.DataFrame.to_parquet()` kwargs test demonstration, though I ended up needing to remove the flexibility downstream within `output`. Pandas with a PyArrow-based `to_parquet` implementation throws errors when unrecognized kwargs are passed. Either way, we should remove the `engine` argument here.

Reply:
Just a note here: Engine was removed as this kwarg is no longer available to be used within the output function.